storybook/src/server/lib/symfony.ts

```diff
@@ -119,34 +119,53 @@ export const getTwigComponentConfiguration = async () => {
 };
 
 export type TwigComponentConfiguration = {
-    anonymousTemplateDirectory: string;
+    anonymousTemplateDirectory: [string];
     namespaces: {
-        [p: string]: string;
+        [p: string]: [string];
     };
 };
 
+type SymfonyTwigConfiguration = {
+    twig: {
+        paths: {
+            [p: string]: string;
+        };
+    };
+};
+
+export const getTwigConfiguration = async () => {
+    return (await runSymfonyCommandJson<SymfonyTwigConfiguration>('debug:config', ['twig', '--resolve-env']))['twig'];
+};
+
+export type TwigConfiguration = {
+    paths: string[];
+};
+
 /**
  * Attempt to resolve the Twig template path containing sources for the given TwigComponent.
  */
 export function resolveTwigComponentFile(componentName: string, config: TwigComponentConfiguration) {
```

Comment:
Can you remove this function? It's not used anymore and has been replace by `TwigComponentResolver`

Reply:
done